src/esn.calendar.libs/app/app.constants.js

```diff
@@ -332,6 +332,11 @@
      */
     .constant('CAL_ALARM_MODIFY_COMPARE_KEYS', ['action', 'attendee', 'description', 'summary', 'trigger'])
 
+    /**
+     * When duplicating an event, the JSON keys on the event will be copied.
+     */
+    .constant('CAL_EVENT_DUPLICATE_KEYS', ['attendees', 'title', 'start', 'end', 'location', 'description', 'alarm', 'xOpenpaasVideoconference'])
```

Comment:
What about resources and privacy?

Reply:
thanks for spotting that, we copy the `privacy` now.

for the `resources` AFAIK they are not actually bound to the event object.